tests/coverage_llvm-cov/test_llvm-cov.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -eu
+# for info see 'read.me'
+
+DIR_BUILD=$PWD
+DIR_SRC=$(cd $(dirname "$0")/../..; pwd -P)
```

Comment:
what will happen if I decide to create build in /home/username/random/dir1/dir2/build but I have source in /home/username/projects/git/repositories/facelift/unpacked/sources ?